tables/output_tables.Rmd

```diff
@@ -0,0 +1,165 @@
+---
+title: "Tables output for manuscript"
+author: "Run Jin"
+date: "10/27/2021"
+output:
+  pdf_document: default
+  html_document: default
+---
+
+```{r setup, include=FALSE}
+
+library(tidyverse)
+
+```
+
+## Output Tables for OpenPBTA Manuscript
+This is a Rmd files that record scripts for generating tables 
+
+```{r define directories}
+root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))
+working_dir <- file.path(root_dir, "tables")
+data_dir <- file.path(root_dir, "data")
+
+results_dir <- file.path(working_dir, "results")
+if(!dir.exists(results_dir)){
+  dir.create(results_dir, recursive=TRUE)
+}
+```
+
+## Table 1: Molecular subtypes determined for this project
+```{r table 1}
+histology_df <- readr::read_tsv(file.path(data_dir, "pbta-histologies.tsv"), guess_max =10000)
+
+subtypes <- histology_df %>%
+  dplyr::filter(!is.na(pathology_diagnosis) & !is.na(molecular_subtype) & !grepl("To be classified", molecular_subtype))
+subtype_table <- subtypes %>%
+  dplyr::group_by(broad_histology, molecular_subtype) %>%
+  tally() %>%
+  readr::write_tsv(file.path(results_dir, "subtypes-n.tsv"))
+
+```
+
+## Table S1: V21 histologies table
+
+```{r v21 histology as excel}
+
+openxlsx::write.xlsx(histology_df, 
+                     file.path(results_dir, "OpenPBTA_histologies_v21.xlsx"),
+                     overwrite=TRUE)
+
+```
+
+## Table S2: DNA results table
+### TMB
+```{r s2 DNA results part 1}
+
+# read in tmb all file, select and rename columns
+tmb_all <- readr::read_tsv(file.path(data_dir, "tcga-snv-mutation-tmb-all.tsv")) %>%
+  dplyr::select(Tumor_Sample_Barcode, tmb) %>%
+  dplyr::rename(Tmb_all = tmb)
+
+# read in tmb coding file, select and rename columns
+tmb_coding <- readr::read_tsv(file.path(data_dir, "tcga-snv-mutation-tmb-coding.tsv")) %>%
```

Comment:
```suggestion
tmb_coding <- readr::read_tsv(file.path(data_dir, "pbta-snv-mutation-tmb-coding.tsv")) %>%
```
This should be the PBTA file instead of TCGA, sorry my mistake!